Sources/Commands/PackageCommands/APIDiff.swift

```diff
@@ -173,39 +194,180 @@ struct APIDiff: AsyncSwiftCommand {
         }
     }
 
-    private func determineModulesToDiff(packageGraph: ModulesGraph, observabilityScope: ObservabilityScope) throws -> Set<String> {
+    private func runWithIntegratedAPIDigesterSupport(_ swiftCommandState: SwiftCommandState, baselineRevision: Revision, baselineDir: Basics.AbsolutePath, modulesToDiff: Set<String>) async throws {
```

Comment:
suggestion: consider type aliasing, or creating more specific type, for the `modulesToDiff` argument to aid in code readability.  For example, instead of `String`, create a type alias which represents what the string is.  e.g.: `typealias ModuleName = String`, and then `modulesToDiff: Set<ModuleName>`